fe/fe-core/src/main/java/org/apache/doris/nereids/rules/implementation/AggregateStrategies.java

```diff
@@ -140,6 +143,71 @@ public List<Rule> buildRules() {
                         return pushdownCountOnIndex(agg, project, filter, olapScan, ctx.cascadesContext);
                     })
             ),
+            RuleType.STORAGE_LAYER_AGGREGATE_MINMAX_ON_UNIQUE_WITHOUT_PROJECT.build(
+                logicalAggregate(
+                        logicalFilter(
+                                logicalOlapScan().when(this::isUniqueKeyTable))
+                                .when(filter -> {
+                                    if (filter.getConjuncts().size() != 1) {
+                                        return false;
+                                    }
+                                    Expression childExpr = filter.getConjuncts().iterator().next().children().get(0);
+                                    if (childExpr instanceof SlotReference) {
+                                        String exprName = ((SlotReference) childExpr).getName();
+                                        return "__DORIS_DELETE_SIGN__".equalsIgnoreCase(exprName);
```

Comment:
u should use `getColumn` and use inferface on Column to check it is delete sign or not.
```
Column#isDeleteSignColumn
```